Makefile

```diff
@@ -12,50 +12,100 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-BINS = $(wildcard velero-*)
+# The binary to build (just the basename).
+# BIN ?= $(wildcard velero-*)
+BIN ?= $(wildcard velero-*)
 
-REPO ?= github.com/heptio/velero-plugin-example
+# This repo's root import path (under GOPATH).
+PKG := github.com/heptio/velero-plugin-example
 
 BUILD_IMAGE ?= gcr.io/heptio-images/golang:1.9-alpine3.6
```

Comment:
we should probably update this while we're here - velero uses `golang:1.11-alpine3.8` (no heptio-images)